jetty-http/src/main/java/org/eclipse/jetty/http/HttpURI.java

```diff
@@ -73,9 +73,16 @@
         /**
          * URI contains ambiguous path parameters within a URI segment e.g. {@code /foo/..;/bar}
          */
-        PARAM
+        PARAM,
+
+        /**
+         * Contains UTF16 encodings
+         */
+        UTF16
     }
 
+    EnumSet<Violation> AMBIGUOUS = EnumSet.complementOf(EnumSet.of(Violation.UTF16));
```

Comment:
Is the name `AMBIGUOUS` here still valid?

Reply:
In fact, is this EnumSet even used? (I don't see it)

Reply:
It is still the right name, as it is the subset that is ambiguous.  It was used in a version... but not now... Unless there is an efficient way to get the intersection of 2 enum sets

Reply:
To work out if we have an ambiguous violation we do
```java
!_violations.isEmpty() && !(_violations.size() == 1 && _violations.contains(Violation.UTF16))
```

So we never use this `AMBIGUOUS` except in testing. Should I inline its usage and remove it?